### PR TITLE
Mkhine/frontenddev

### PR DESCRIPTION
**Pull request by MayKhine**
Pls review

### CODE REVIEW

File: packages/frontend/src/App.tsx [hunk 1]
```diff
@@ -2,43 +2,73 @@
...
+
+  //       { character: "", editable: true },
+  //     ],
+  //     [
+  //       { character: "", editable: true },
+  //       { character: "", editable: true },
+  //     ],
+  //     [{ character: "A", editable: false }],
+  //   ],
+  // };
```
Comment (EthanSherr) [inline packages/frontend/src/App.tsx:41]: can you remove this unused stuffs

File: packages/frontend/src/App.tsx [hunk 2]
```diff
@@ -2,43 +2,73 @@
...
 
   return (
     <div>
-      <Pyramid data={data} />
+      <div {...stylex.props(styles.logo)}>WORD PYRAMID</div>
+      <div {...stylex.props(styles.pyramidDiv)}>
+        {" "}
+        {data ? <Pyramid pyramidData={data} /> : "loading"}
+      </div>{" "}
+      {/* <Pyramid pyramidData={data} /> */}
```
Comment (EthanSherr) [inline packages/frontend/src/App.tsx:50]: more unused remove

File: packages/frontend/src/App.tsx [hunk 3]
```diff
@@ -2,43 +2,73 @@
...
+    minWidth: "225px",
+    alignSelf: "center",
+    height: "3.5rem",
+    // backgroundColor: "pink",
+    // display: 'flex'
+  },
+  pyramidDiv: {
+    // height: "calc(100% - 6.5rem) ",
+    // backgroundColor: "pink",
+  },
```
Comment (EthanSherr) [inline packages/frontend/src/App.tsx:73]: unused thingy?

File: packages/frontend/src/components/Button.tsx [hunk 4]
```diff
@@ -23,22 +30,26 @@ export const Button = ({ text, type, onClickFn, width }: ButtonProps) => {
 
 const styles = stylex.create({
   base: (props) => ({
-    backgroundColor: props.text === "submit" ? tokens.green : tokens.yellow,
+    // backgroundColor: props.text === "submit" ? tokens.green : tokens.yellow,
```
Comment (EthanSherr) [inline packages/frontend/src/components/Button.tsx:33]: 1 more

File: packages/frontend/src/components/Button.tsx [hunk 5]
```diff
@@ -23,22 +30,26 @@ export const Button = ({ text, type, onClickFn, width }: ButtonProps) => {
...
+    backgroundColor: props.bgColor,
 
     minWidth: "10rem",
     // width: "80%",
     width: props.width && props.width.length > 0 ? props.width : "80%",
     maxWidth: "40rem",
-    fontSize: "2rem",
-    height: "5rem",
+    fontSize: "2em",
+    // height: "5rem",
```
Comment (EthanSherr) [inline packages/frontend/src/components/Button.tsx:41]: 1 more

File: packages/frontend/src/components/Pyramid.tsx [hunk 6]
```diff
@@ -156,13 +126,36 @@ export const Pyramid = ({ data }: PyramidType) => {
     }
   };
 
+  const [shake, setShake] = useState(false);
+  // const [emojiPosition, setEmojiPosition] = useState({ x: 0, y: 0 });
```
Comment (EthanSherr) [inline packages/frontend/src/components/Pyramid.tsx:130]: remove

File: packages/frontend/src/App.tsx (hunk 1 above)
Comment (MayKhine) [reply packages/frontend/src/App.tsx:41]: ok

**Pull request merged**